client/battle/BattleActionsController.h

```diff
@@ -53,7 +53,7 @@ class BattleActionsController
 	bool isCastingPossibleHere (const CSpell * spell, const CStack *shere, BattleHex myNumber);
 	bool canStackMoveHere (const CStack *sactive, BattleHex MyNumber) const; //TODO: move to BattleState / callback
 	std::vector<PossiblePlayerBattleAction> getPossibleActionsForStack (const CStack *stack) const; //called when stack gets its turn
-	void reorderPossibleActionsPriority(const CStack * stack, MouseHoveredHexContext context);
+	void reorderPossibleActionsPriority(const CStack * stack, const CStack * targetStack);

```

Comment:
Am I correct that this was the only usage of `MouseHoveredHexContext`? If yes - enum should probably be removed as well